specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.1/Prediction.json

```diff
@@ -12,11 +12,8 @@
   "paths": {
     "/{projectId}/classify/iterations/{publishedName}/image": {
       "post": {
-        "tags": [
-          "ImagePredictionApi"
-        ],
-        "summary": "Classify an image and saves the result.",
-        "operationId": "ClassifyImage",
+        "description": "Classify an image and saves the result.",
+        "operationId": "ImageClassification_Image",
```

Comment:
Guidelines suggest Noun_Verb because the generated SDK will use the Noun as the class name and Verb as method name.  Suggest `ImageClassifier_ClassifyImage`.